src/engine/platform/opl.h

```diff
@@ -194,6 +208,10 @@ class DivPlatformOPL: public DivDispatch {
     void renderSamples(int chipID);
     int init(DivEngine* parent, int channels, int sugRate, const DivConfig& flags);
     void quit();
+    YMF278& getChip();
```

Comment:
Reason?

Reply:
Leftover from PR https://github.com/tildearrow/furnace/pull/387, It's unused. I will delete this.